tedana/decomposition/ica.py

```diff
@@ -1,30 +1,48 @@
-"""
-ICA and related signal decomposition methods for tedana
-"""
+"""ICA and related signal decomposition methods for tedana"""
```

Comment:
I can't say definitively what will need to happen here, specifically because I think some of the code formatting in this regard may have changed upstream. But as general advice: As a software maintainer, I don't want an incoming proposed change from an external contributor to include a bunch of code formatting changes that are unrelated to the actual functional change being proposed. If there's a recommendation for some kind of formatting change, eg. based on some standard, or maybe even to make formatting throughout that code base more self-consistent, I would prefer that that be proposed as a stand-alone PR.

Reply:
Consistent formatting was applied through commit 4fc3043. Used the tedana's preferred style.

Reply:
So there's multiple things happening here that I need to walk through.

This specific comment is tied to a specific proposed code change, which was a part of the PR at time of posting. That proposed change is a modification of the formatting of a comment that already existed in TEDANA. So my comment was specifically about the modification of formatting of existing code, not the format style to be used for new code additions. Your linked commit therefore doesn't actually address the question raised: while you do apply the formatting standard that is used elsewhere in TEDANA, it does so exclusively for new code. Indeed the specific code comment to which this GitHub comment is attached is unmodified by your linked commit.

What I think may have happened here is that during the timespan of the evolution of this PR, there have been changes upstream at TEDANA relating to this formatting. Because I then had to do a bunch of git gymnastics to restructure your proposed contribution---including omission of merges between your feature branch and upstream---that may have led to a mismatch of what formatting style is used where.

The key point is that the comprehensive state of what is proposed in the PR should not include inconsequential formatting changes that are not relevant to your proposed feature addition. At time of writing that is the case. You can also see in GitHub that the code change to which the discussion is attached has an "Outdated" flag.